rllib/agents/slateq/slateq.py

```diff
@@ -78,16 +79,20 @@
     "huber_threshold": 1.0,
 
     # === Replay buffer ===
-    # Size of the replay buffer. Note that if async_updates is set, then
-    # each worker will have a replay buffer of this size.
-    "buffer_size": DEPRECATED_VALUE,
     "replay_buffer_config": {
-        "type": "MultiAgentReplayBuffer",
+        # Enable the new ReplayBuffer API.
+        "_enable_replay_buffer_api": True,
```

Comment:
is there a reason that you have to set this and its not just the case already? 

Does this make Slateq use another replay buffer api?

Reply:
Yeah, it'll use the new replay buffer API we are currently rolling out across RLlib.